lib/utils/tagGenarator.js

```diff
@@ -0,0 +1,17 @@
+const TagModelManager = require('../../lib/modelManagers/tagModel');
+const { ArticleTags } = require('../../database/models');
+
+const generateTags = async (tag, id) => {
+  const tags = tag.split(',');
+  const keys = Array.from(Object.values(tags));
```

Comment:
`tag.split(',');` returns an array. Why are you still perform this ` Array.from(Object.values(tags));` operation?

Reply:
i have fixed this.